src/gtest/main.cpp

```diff
@@ -24,6 +26,8 @@ int main(int argc, char **argv) {
   boost::filesystem::path pk_path = ZC_GetParamsDir() / "sprout-proving.key";
   boost::filesystem::path vk_path = ZC_GetParamsDir() / "sprout-verifying.key";
   params = ZCJoinSplit::Prepared(vk_path.string(), pk_path.string());
+
+  TestSaplingParams = librustzcash_init_params();
```

Comment:
This confused me for a moment. Could we rename it to `pTestSaplingParams`, or some other name that doesn't look like a class?